doc/loanrules.md

```diff
@@ -33,12 +39,14 @@ until the end of the line are ignored.
 A name of a loan policy, a patron group, a material type, a loan type or a location
 can contain only these characters: a-z, A-Z, 0-9 and minus "-".
 
-## Loan Policy
+## Policies
 
-The name of a loan policy is appended to a criteria line separated by a colon.
-If the line matches then that policy is applied.
+A list of policies is appended to a criteria line after a colon, with each policy's
+type indicated by an `l`, `r`, or `n`.
+They can be in any order, but must have one of each type.
```

Comment:
Will there always be all 3 policy types after the colon, or is it possible to leave out 1 or 2 of them?
Filip Jakobsen answered: Any number of policies can be defined

Reply:
Example from discussion with Filip Jakobsen:

```
fallback-policy: l does-not-circulate r non-requestable n general-non-circ

g staff: l long_loan r long_loan_req n long_loan_notice
    s new_items: l short_loan_fixed
```

There should be an example in this loanrules.md showing a rule with 1 or 2 policies, and a test case in the unit tests with 1 or 2 policies.

Reply:
I think we should create a new issue to address this. When a rule is generated you only have the PoliciesContext to work with, and I would have to find a way to access the FallbackPoliciesContext and substitute the missing policies.

Since this is a high priority card that is blocking several UI stories it might be better to move this work forward without this change and have me start this work in parallel with the front-end work. The other items seem to be easily fixed so I can take care of them now.

Reply:
@julianladisch Can you refer me to where Filip has made that decision? We likely need to loop Cate and Sean into that conversation.

Does this only apply to individual policy lines? Or fallback-policy lines as well? (as that would be different to my interpretation of your comment on https://issues.folio.org/browse/CIRC-196?focusedCommentId=44019&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-44019)

Reply:
I've asked this question on the related UI issue - https://issues.folio.org/browse/UICIRC-164?focusedCommentId=44296&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-44296

Please feel free to correct my understanding if I am incorrect.

Reply:
The fallback policy always have all three policy types to ensure that always a policy will be returned. Only the rule lines may have less than 3 policy types.

Reply:
Regarding the example I posted above 3 hours ago:
On slack I asked: "What is the request and the notice policy for an item from new_items for staff?"
Filip Jakobsen answered: "it inherits from its parent (i.e. "g staff"), so they are *r long_loan_req* and *n long_loan_notice*"

Reply:
@julianladisch Which slack conversation was this question asked and answered by @filipjakobsen

Reply:
This was on a three person slack conversation (Sean Thomas, Filip Jakobsen, Julian Ladisch), the same where the "Circulation Rules Editor Questions - Q1 2019 - Julian/Filip" document was discussed.